cli/http_util.rs

```diff
@@ -75,7 +76,7 @@ fn resolve_url_from_location(base_url: &Url, location: &str) -> Url {
 #[derive(Debug, PartialEq)]
 pub enum FetchOnceResult {
   // (code, maybe_content_type, etag)
```

Comment:
update this comment. It might make sense use a struct here since the number of fields is now a lot.

Reply:
Moved to a struct, it was getting a bit silly.